src/components/TableColumnSetup/index.ts

```diff
@@ -0,0 +1 @@
+export * from './TableColumnSetup';
```

Comment:
Should export it also in https://github.com/gravity-ui/uikit/blob/645df5615cc52f419cd7cb57865af454ba1ad8be/src/components/index.ts

Reply:
bcde4ae403818d7103a6bac8874b87ba08d2f7da